adyenofficial.php

```diff
@@ -1011,71 +1108,108 @@ public function displayForm()
             'required' => true
         );
 
```

Comment:
I'd separate the settings with comments just to make it a bit easier to read and understand the code later (also for the other sections)

```suggestion
// Production settings
```

Reply:
https://github.com/Adyen/adyen-prestashop/pull/203/commits/518f6c41b9dd8780fb9af5e666c67f2f5afcd24e